contracts/Account.sol

```diff
@@ -7,13 +7,14 @@ import "./Managed.sol";
 import "./common/UUPSOwnableUpgradeable.sol";
 import "./common/UsingRegistryUpgradeable.sol";
 import "./interfaces/IAccount.sol";
+import "./Pausable.sol";
```

Comment:
I believe that all important methods are covered. We could consider introduction of pausable granularity. eg. pause only voting, deposits, withdrawals